### ISSUE TITLE
[clox] Common constant instructions all increment the line number.

### ISSUE THREAD

**Issue by Dandigit** [Dandigit/pl-projects#4]
This is a weird one.
`OP_TRUE`, `OP_FALSE` and `OP_NIL` all increment the line number of the instruction after it.

Here's some sample output:
```
lox > nil
== code ==
0000    1 OP_NIL
0001    2 OP_RETURN
          
0000    1 OP_NIL
          [ nil ]
0001    2 OP_RETURN
nil
```
After `nil`, the line number is incremented.

```
lox > true == true
== code ==
0000    1 OP_TRUE
0001     | OP_TRUE
0002     | OP_EQUAL
0003    2 OP_RETURN
          
0000    1 OP_TRUE
          [ true ]
0001     | OP_TRUE
          [ true ][ true ]
0002     | OP_EQUAL
          [ true ]
0003    2 OP_RETURN
true
```
This one is really strange. `OP_EQUAL` and the other `OP_TRUE` are on the same line but `OP_RETURN` is on the other line for some reason.

**Comment by Dandigit** [comment]
OK, it seems that only if the common constant instruction is the first instruction compiled, this issue occurs. Still don't know why.

**Comment by Dandigit** [comment]
Fixed in #5.
